common/changes/office-ui-fabric-react/master_2018-07-25-11-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "VSO: 603037:- Either render the initials or the avtar image (PersonaCoin) and not both in the site permission panel",
```

Comment:
The comment of the change is very specific to the application it is being integrated into. It should be more generic with focus on what it changes in fabric-react instead